### ISSUE TITLE
Running two instances of Pokeminer and feeding data to the same map?

### ISSUE THREAD

**Issue by gatomiaurex** [modrzew/pokeminer#242]
So with Niantic banning accounts left and right I found myself with 30 out of my 74 accounts banned so I want to know if I can like

1.- run the workers and the servers covering 1 part of my city
2.- copy the pokeminer folder
3.- run another instance of workers running another part of my city

Basically my city has an irregular shape and some of the workers are scanning empty things so I am wasting workers and this is no time to do that.

![untitled](https://cloud.githubusercontent.com/assets/20795984/17712223/4c8edeb6-63b2-11e6-9792-3f4ce2b86b50.png)

**Comment by amistad18** [comment]
Check this out - https://github.com/modrzew/pokeminer/pull/100 , and here fresh PR https://github.com/modrzew/pokeminer/pull/238 ... there isn't much changes there, so you will be able to do them manually. Also remember to update config file to set disabled workers there.

**Comment by gatomiaurex** [comment]
I tried that a while ago and it only disables the workers and doesn't make them available for scanning another locations, am I right? Or maybe I misunderstood the commit?

**Comment by modrzew** [comment]
@gatomiaurex yes, that's possible. You can have any number of pokeminers feeding the same database. Be sure to use MySQL instead of SQLite, though.

You can also define a large rectangle and disable some of the workers using PRs @amistad18 mentioned.

**Comment by gatomiaurex** [comment]
I was about to try that but as soon as I opened the workers 7 more accounts got banned, I am using 10 steps per worker and 12 scan delay, did Niantic changed something? The accounts are getting banned faster than before. :/

**Comment by r0bodog** [comment]
How would I go about changing to MySQL from your code?
